spark/src/main/scala/io/delta/sharing/spark/RemoteDeltaCDFRelation.scala

```diff
@@ -111,6 +126,33 @@ object DeltaSharingCDFReader {
       removeFiles.map(r => r.id -> r.url).toMap
   }
 
+  def getMinUrlExpiration(
+      addFiles: Seq[AddFileForCDF],
+      cdfFiles: Seq[AddCDCFile],
+      removeFiles: Seq[RemoveFile]
+  ): Long = {
+    var minUrlExpiration: Long = Long.MaxValue
```

Comment:
Can we use Option[Long] and use `Some(v)` to indicate that we know the expiration timestamp, and None to indicate that we don't know the expiration time rather than -1 or Long.MaxValue?

Otherwise, for each place using the expiration time, we have to think about three cases: a valid value, -1 and Long.MaxValue

Reply:
sg, will make the change.